src/Type/Definition/ListOfType.php

```diff
@@ -22,12 +22,7 @@ public function toString() : string
         return '[' . $this->ofType->toString() . ']';
     }
 
-    /**
-     * @param bool $recurse
-     *
-     * @return ObjectType|InterfaceType|UnionType|ScalarType|InputObjectType|EnumType
```

Comment:
I'm not sure but in some places this annotation makes sense as Type is too wide. I'll look at it more deeply later.

Reply:
Let me know if you can figure it out--in the meantime, I did the same basic `Type` conversion on `ofType` and was able to add a typehint and remove an assert, which is nice.

Reply:
> it returns any Type except instances of WarppingType

Is that true? Couldn't the `ofType` be another `ListOfType`?